templates/micropub-writing-settings.php

```diff
@@ -1,10 +0,0 @@
-<fieldset id="micropub">
```

Comment:
in the future, if you move a file, you want to use `git mv` so that git records the move, so you can see diffs, follow history across filenames, etc.

Reply:
I git mved the wrong file.